### ISSUE TITLE
picture through MMS --> Failed!

### ISSUE THREAD

**Issue by Thieb67k** [ubports/ubuntu-touch#976]
- Device: LGE Nexus 4
- Channel: stable
- Build: OTA-5

**Comment by UniversalSuperBox** [comment]
Hello @Thieb67k, can you provide more information with this issue? I have no idea what the issue is or how to reproduce it. Please follow [Contribute to Ubuntu Touch - Bug reporting](https://docs.ubports.com/en/latest/contribute/bugreporting.html) to provide the relevant information.

If no further information is provided, this report will be closed on December 10.

**Comment by Thieb67k** [comment]
For unknown reason my description did not appears.
Hope that can help people to solve the problem.
:)

**Comment by vlladdrakk** [comment]
I have this same issue on my Nexus 4 on the stable channel (OTA-5) when I have both mobile data and wifi on. When I turn wifi off I can send a picture MMS.

**Comment by Thieb67k** [comment]
Unfortunately, even if wifi is off the picture MMS is not sent.

**Comment by Thieb67k** [comment]
That bug still happens with OTA-6 on LGE Nexus 4.

**Comment by Thieb67k** [comment]
Hello,
It seems to be similar to #12 and #276  .

**Comment by Thieb67k** [comment]
Picture+MMS still not possible with OTA-7.

**Comment by dobey** [comment]
Can you please specify your carrier? It sounds like perhaps there may be an issue with the APN config for your carrier.

**Comment by hummlbach** [comment]
+1 for dobeys assumption. I had to set proxy and proxy port manually.

**Comment by Thieb67k** [comment]
True, I have seen I can enable MMS by use of "mobile data" + "data roaming".
That was not the case with my previous mobile OS.
Problem is solved, but expensive now.

**Comment by hummlbach** [comment]
@Flohack74 i would suggest to close this. What do you think?

**Comment by Flohack74** [comment]
I would not. MMS is a problem for some users, and we still don´t know why. We also need to do this to be sure people can configure everything they need: https://github.com/ubports/system-settings/issues/134

**Comment by osimmasgard** [comment]
I have also had this issue with my FP2 2019-W09, and have tried messing around with the APN settings **a lot**. I just enabled roaming and it started working.

**Comment by Flaburgan** [comment]
Same than @osimmasgard here, with SFR, I configured the SFR APN as told [there](https://assistance.sfr.fr/tel-mobile/utilisation-param/parametrer-apn-sfr-mobile.html), and I can only send MMS, not receive them. I tried to have 2 different configurations, one for MMS and one for internet, but in that case I wasn't even able to send them. 

I just saw that there is also #276 and #12 open, maybe those more recent issue should be closed as duplicates and only #12 should be kept?
